docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,311 @@
+# Bootstrapping Trust
+
+## Part 1: Encryption Everywhere
+
+Whether you are an encryption expert or a newcomer, welcome! This series
+is for you! It assumes you know nothing and takes you from soup to nuts
+on how to bootstrap trust with the intent to power a Zero Trust security
+model. The process and thinking described in this series are the direct
+output of developing the same system for the Ziti open source project.
+Ziti can be found on the GitHub project page for
+[OpenZiti](https://github.com/openziti). The series starts with the
+basics and dovetails into Ziti's Enrollment system.
+
+The parts are as follows.
+
+- [Part 1: Encryption Everywhere](./part-01.encryption-everywhere.md)
+- [Part 2: A Primer On Public-Key Cryptography](./part-02.a-primer-on-public-key-cryptography.md)
+- [Part 3: Certificates](./part-03.certificates.md)
+- [Part 4: Certificate Authorities & Chains Of Trust](./part-04.certificate-authorities-and-chains-of-trust.md)
+- [Part 5: Bootstrapping Trust](./part-05.bootstrapping-trust.md)
+
+
+### Zero Trust
+
+This entire series assumes some familiarity with Zero Trust. If you do
+not have a strong background in it, that is fine. This section should
+give the reader enough context to make use of the entire series. If a
+more in-depth understanding is desired, please consider reading Zero
+Trust Networks: Building Secure Systems in Untrusted Networks by Evan
```

Comment:
apparently titles of books get italicized ... who knew (just consulted my resident expert) :D

Reply:
done